src/middlewared/middlewared/api/v25_04_0/user.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Optional
+
+from annotated_types import Ge, Le
+from pydantic import EmailStr
+from typing_extensions import Annotated
+
+from middlewared.api.base import *
```

Comment:
Doing this type of import is discouraged for numerous reasons. It can be expensive since this module now has to import everything in `api.base`. It also runs risk of causing namespace collisions. Overall it's just bad practice.